lib/connect-session-sequelize.js

```diff
@@ -23,16 +23,14 @@ util.inherits(SequelizeStoreException, Error);
 
 module.exports = function SequelizeSessionInit(Store) {
   function SequelizeStore(options) {
-    options = options || {};
+    this.options = options = options || {};
     if (!options.db) {
       throw new SequelizeStoreException('Database connection is required');
     }
 
-    var key;
-    this.options = defaultOptions;
-    for (key in options) {
-      if (key !== 'db') {
-        this.options[key] = options[key];
+    for (var key in defaultOptions) {
+      if (key in options === false) {
```

Comment:
yeah `key in options` would return `true` if `options[key]` is set, regardless if the value is truthy or falsy. it also returns `true` when a prototype of the object has this value, thus one could argue that it might be better to use `options.hasOwnProperty(key)`, in case one wouldn't want to the values' of an object's prototype.

we could also use underscore's or lodash's `_.defaults(options, defaultOptions)`.